packages/mobile/src/screens/chat-screen/ChatMessageListItem.tsx

```diff
@@ -199,6 +159,21 @@ export const ChatMessageListItem = forwardRef<View, ChatMessageListItemProps>(
       y: number
     }>({ x: 0, y: 0 })
 
+    const reactionPositionStyle = [
```

Comment:
does this not get inferred if the bubble is marked as `position: relative`?

Reply:
On web what I did was basically set the `right` or `left` styles relative to the parent based on `isAuthor`, then changed text direction (in your case, could be just reversing the container list: `flexDirection: 'row-reverse'` or `flexDirection: 'row'`)

Reply:
Basically if we can avoid using state to get positions of the parent by inferring using position relative, that would be good, and then also if we can get rid of manually creating styles within the component and have two "classes" instead

Reply:
Amazing! I think I tried something similar but was having trouble with position the tail when the reaction container position was anything other than absolute. But I realized I can set the height of the container to 0 and it'll still be visible.